docs/src/main/asciidoc/kubernetes-resources.adoc

```diff
@@ -124,3 +124,18 @@ The full source of the `kubernetes.json` file looks something like this:
 An important thing to note about the `Deployment` is that is uses `yourDockerUsername/test-quarkus-app:1.0-SNAPSHOT` as the Docker image of the `Pod`.
 
 Also the `Service` is configured to use container port `8080` (which is automatically picked up by the standard Quarkus configuration).
+
+=== Add readiness and liveness probes
+
+By default the Kubernetes resources do not contain readiness and liveliness probes in the generated `Deployment`. Adding them however is just a matter of adding the Smallrye Health extension like so:
```

Comment:
```suggestion
By default the Kubernetes resources do not contain readiness and liveness probes in the generated `Deployment`. Adding them however is just a matter of adding the Smallrye Health extension like so:
```
I think they are called liveness probes. At least that's what we use in the configuration property.

Reply:
Yup, good catch, I had changed some but missed this one. Fixed now